[0.32.0-RC2] liquibase.exception.ValidationFailedException: Validation Failed: migrations/000_migrations.yaml

I just tried to upgrade to 0.32.0-RC2 from 0.31.2 and got this error ([full log](https://github.com/metabase/metabase/files/3006835/log.txt)):
```
ERROR metabase.core :: Metabase Initialization FAILED
liquibase.exception.ValidationFailedException: Validation Failed:
     1 change sets check sum
          migrations/000_migrations.yaml::10::cammsaul was: 8:5297214d1788d964675d8c6336ac9b6d but is now: 8:96e54d9100db3f9cdcc00eaeccc200a3
```
Metabase 0.31.2 JAR with internal MariaDB on Ubuntu 16.04
@camsaul Do you need anything else? I reverted to 0.31.2, but can try again tomorrow night.

I ran into the same issue today. First I tried to force the migration:

`java -jar metabase-v0.32.0-RC2.jar migrate force`

After that I tried to start the server and ran into another issue:

```
03-27 07:48:16 ERROR changelog.ChangeSet :: Change Set migrations/000_migrations.yaml::99::camsaul failed.  Error: (conn=764026) Duplicate key name 'idx_uniq_field_table_id_parent_id_name' [Failed SQL: ALTER TABLE mbasedb.metabase_field ADD CONSTRAINT idx_uniq_field_table_id_parent_id_name UNIQUE (table_id, parent_id, name)]
03-27 07:48:16 WARN metabase.util :: auto-retry metabase.db$run_schema_migrations_BANG_$fn__14466@3d072a74: Migration failed for change set migrations/000_migrations.yaml::99::camsaul:
     Reason: liquibase.exception.DatabaseException: (conn=764026) Duplicate key name 'idx_uniq_field_table_id_parent_id_name' [Failed SQL: ALTER TABLE mbasedb.metabase_field ADD CONSTRAINT idx_uniq_field_table_id_parent_id_name UNIQUE (table_id, parent_id, name)]
03-27 07:48:16 INFO metabase.db :: Setting up Liquibase...
03-27 07:48:16 INFO metabase.db :: Liquibase is ready.
03-27 07:48:16 INFO metabase.db :: Checking if Database has unrun migrations...
03-27 07:48:17 INFO metabase.db :: Database has unrun migrations. Waiting for migration lock to be cleared...
03-27 07:48:17 INFO metabase.db :: Migration lock is cleared. Running migrations...
03-27 07:48:17 ERROR changelog.ChangeSet :: Change Set migrations/000_migrations.yaml::99::camsaul failed.  Error: (conn=764028) Duplicate key name 'idx_uniq_field_table_id_parent_id_name' [Failed SQL: ALTER TABLE mbasedb.metabase_field ADD CONSTRAINT idx_uniq_field_table_id_parent_id_name UNIQUE (table_id, parent_id, name)]
03-27 07:48:17 ERROR metabase.core :: Metabase Initialization FAILED
liquibase.exception.MigrationFailedException: Migration failed for change set migrations/000_migrations.yaml::99::camsaul:
     Reason: liquibase.exception.DatabaseException: (conn=764028) Duplicate key name 'idx_uniq_field_table_id_parent_id_name' [Failed SQL: ALTER TABLE mbasedb.metabase_field ADD CONSTRAINT idx_uniq_field_table_id_parent_id_name UNIQUE (table_id, parent_id, name)]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:637)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:53)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:83)
	at liquibase.Liquibase.update(Liquibase.java:202)
	at liquibase.Liquibase.update(Liquibase.java:179)
	at liquibase.Liquibase.update(Liquibase.java:175)
	at metabase.db$migrate_up_if_needed_BANG_.invokeStatic(db.clj:209)
	at metabase.db$migrate_up_if_needed_BANG_.invoke(db.clj:196)
	at metabase.db$migrate_BANG_$fn__14448.invoke(db.clj:313)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:799)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:769)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:834)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:769)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:782)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:769)
	at metabase.db$migrate_BANG_.invokeStatic(db.clj:301)
	at metabase.db$migrate_BANG_.invoke(db.clj:282)
	at metabase.db$run_schema_migrations_BANG_$fn__14466.invoke(db.clj:446)
	at metabase.util$do_with_auto_retries.invokeStatic(util.clj:428)
	at metabase.util$do_with_auto_retries.invoke(util.clj:420)
	at metabase.util$do_with_auto_retries.invokeStatic(util.clj:432)
	at metabase.util$do_with_auto_retries.invoke(util.clj:420)
	at metabase.db$run_schema_migrations_BANG_.invokeStatic(db.clj:445)
	at metabase.db$run_schema_migrations_BANG_.invoke(db.clj:428)
	at metabase.db$setup_db_BANG_$fn__14475.invoke(db.clj:468)
	at metabase.util$do_with_us_locale.invokeStatic(util.clj:683)
	at metabase.util$do_with_us_locale.invoke(util.clj:669)
	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:466)
	at metabase.db$setup_db_BANG_.doInvoke(db.clj:460)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at metabase.core$init_BANG_.invokeStatic(core.clj:72)
	at metabase.core$init_BANG_.invoke(core.clj:51)
	at metabase.core$start_normally.invokeStatic(core.clj:118)
	at metabase.core$start_normally.invoke(core.clj:112)
	at metabase.core$_main.invokeStatic(core.clj:138)
	at metabase.core$_main.doInvoke(core.clj:133)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at metabase.core.main(Unknown Source)
Caused by: liquibase.exception.DatabaseException: (conn=764028) Duplicate key name 'idx_uniq_field_table_id_parent_id_name' [Failed SQL: ALTER TABLE mbasedb.metabase_field ADD CONSTRAINT idx_uniq_field_table_id_parent_id_name UNIQUE (table_id, parent_id, name)]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:356)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:125)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1229)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1211)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:600)
	... 39 more
Caused by: java.sql.SQLSyntaxErrorException: (conn=764028) Duplicate key name 'idx_uniq_field_table_id_parent_id_name'
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:236)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.getException(ExceptionMapper.java:165)
	at org.mariadb.jdbc.MariaDbStatement.executeExceptionEpilogue(MariaDbStatement.java:238)
	at org.mariadb.jdbc.MariaDbStatement.executeInternal(MariaDbStatement.java:356)
	at org.mariadb.jdbc.MariaDbStatement.execute(MariaDbStatement.java:407)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:352)
	... 44 more
Caused by: java.sql.SQLException: Duplicate key name 'idx_uniq_field_table_id_parent_id_name'
Query is: ALTER TABLE mbasedb.metabase_field ADD CONSTRAINT idx_uniq_field_table_id_parent_id_name UNIQUE (table_id, parent_id, name)
java thread: main
	at org.mariadb.jdbc.internal.util.LogQueryTool.exceptionWithQuery(LogQueryTool.java:126)
	at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.executeQuery(AbstractQueryProtocol.java:222)
	at org.mariadb.jdbc.MariaDbStatement.executeInternal(MariaDbStatement.java:350)
	... 46 more
```

To solve this issue I had to drop the index and start the server again:

`ALTER TABLE metabase_field DROP INDEX  idx_uniq_field_table_id_parent_id_name;`

As far as I've tested everything is fine, but I would wait until the final version comes out to upgrade in production.

--- a/resources/migrations/000_migrations.yaml
+++ b/resources/migrations/000_migrations.yaml
@@ -1465,12 +1465,14 @@ databaseChangeLog:
       id: 10
       author: cammsaul
       validCheckSum:
-        - 7:97fec69516d0dfe424ea7365f51bb87e
         - 7:3b90e2fe0ac8e617a1f30ef95d39319b
-        - 8:431360d062cb82d8b27960b3a0abb98c
-        - 8:9f03a236be31f54e8e5c894fe5fc7f00
+        - 7:97fec69516d0dfe424ea7365f51bb87e
         - 8:2e03a495932b4a9aebb9d58a6ad87ca9
+        - 8:431360d062cb82d8b27960b3a0abb98c
+        - 8:5297214d1788d964675d8c6336ac9b6d
         - 8:532075ff1717d4a16bb9f27c606db46b
+        - 8:96e54d9100db3f9cdcc00eaeccc200a3
+        - 8:9f03a236be31f54e8e5c894fe5fc7f00
       changes:
         - createTable:
             tableName: revision

